src/tests/JIT/Regression/JitBlue/GitHub_39823/GitHub_39823.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestPriority>1</CLRTestPriority>
+  </PropertyGroup>
+  <PropertyGroup>
+    <DebugType />
```

Comment:
`<DebugType />` I saw this before but it is a useless line or does it do something?

Reply:
I change this to Full to be explicit.